understanding/20/language-of-parts.html

```diff
@@ -141,14 +141,37 @@ <h3>Alternative language links</h3>
+           <code>a</code> element (as in the second example) is more reliable and ensures that screen readers pronounce the content correctly.</p>
+      </div>
+
+      <h3>Element's content in multiple languages used as accessible name</h3>
+      <p>When content includes multiple languages, any child elements that use a different language must have their own <code>lang</code> attribute.</p>
+      <p>This example assumes the page's default language is English. The main text of the link is in English, but it includes the original Italian title of a poem. The Italian portion is correctly marked with lang="it".</p>
+      <p><code class="language-html">&lt;a href="/la-divina-commedia.html"&gt;Read this book: &lt;span lang="it"&gt;La Divina Commedia&lt;/span&gt;&lt;/a&gt;</code></p>
+      <div class="note">
+          <p>Current user agents and assistive technologies have limited and inconsistent support for handling language changes as part of their accessible name calculation and exposition.
+             Since the accessible name of this link contains multiple languages, it is required to mark each portion with the appropriate <code>lang</code>, either inherited or set explicitly,
+             while being aware that assistive technologies may not always render the languages correctly.</p>
+      </div>
 
          <h3>"Podcast" used in a French sentence</h3>
          <p>Because "podcast" is part of the vernacular of the immediately surrounding text in the following excerpt, "<span lang="fr">À l'occasion de l'exposition "Energie éternelle. 1500 ans d'art indien", le Palais des Beaux-Arts de Bruxelles a lancé son premier podcast. Vous pouvez télécharger ce podcast au format M4A et MP3</span>", no indication of language change is required.</p>
 
          <h3>The element's content and attribute values are in different languages</h3>
-         <p>This example assumes that the page's default content is in English. The link's <code class="language-html">title</code> attribute is in English, but the nested <code class="language-html">span</code> element that contains the word <span lang="es">Español</span> has a <code>lang="es"</code> attribute.</p>
+         <p>This example assumes that the page's default content is in English. The link's <code class="language-html">title</code> attribute is in English, but the nested <code class="language-html">span</code> element that contains the words <span lang="it">La Divina Commedia</span> has a <code>lang="it"</code> attribute.</p>
 
-         <p><code class="language-html">&lt;a title="Spanish" href="qa-html-language-declarations-es.html"&gt;&lt;span lang="es"&gt;Español&lt;/span&gt;&lt;/a&gt;</code></p>
+         <p><code class="language-html">&lt;a title="Opens in a new window" href="/la-divina-commedia.html"&gt;Read this book: &lt;span lang="it"&gt;La Divina Commedia&lt;/span&gt;&lt;/a&gt;</code></p>
```

Comment:
```suggestion
         <p><code class="language-html">&lt;a title="Opens in a new tab" href="/la-divina-commedia.html" target="_blank"&gt;Read this book: &lt;span lang="it"&gt;La Divina Commedia&lt;/span&gt;&lt;/a&gt;</code></p>
```
`target` should be added so the `title` is correct. I also assume you meant opening it in a new tab because opening in a new window is uncommon?